crates/cargo-util-schemas/manifest.schema.json

```diff
@@ -621,40 +621,7 @@
         }
       ]
     },
-    "TomlValue": {
-      "type": "object",
-      "properties": {
-        "string": {
-          "type": "string"
-        },
-        "integer": {
-          "type": "integer",
-          "format": "int64"
-        },
-        "float": {
-          "type": "number",
-          "format": "double"
-        },
-        "boolean": {
-          "type": "boolean"
-        },
-        "datetime": {
-          "type": "string"
-        },
-        "array": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/TomlValue"
-          }
-        },
-        "table": {
-          "type": "object",
-          "additionalProperties": {
-            "$ref": "#/definitions/TomlValue"
-          }
-        }
-      }
-    },
+    "TomlValue": true,
```

Comment:
I can't find the definition of this boolean in JSON schema spec, but it seems to accept anything so indeed fixes the issue. Tested it on <https://www.jsonschemavalidator.net/>

Reply:
https://json-schema.org/draft/2020-12/json-schema-core#name-boolean-json-schemas

> The boolean schema values "true" and "false" are trivial schemas that always produce themselves as assertion results, regardless of the instance value. They never produce annotation results.